plugins/woocommerce/src/Blocks/BlockTypes/ProductButton.php

```diff
@@ -82,9 +83,14 @@ protected function render( $attributes, $content, $block ) {
 		$post_id = isset( $block->context['postId'] ) ? $block->context['postId'] : '';
 		$product = wc_get_product( $post_id );
 
+		// Initialize the "Add To Cart" store part.
+		// Question: Is this ok for 3PD or should we use a global function like `woocommerce_interactivity_use_add_to_cart_store()`.
```

Comment:
Question: Is this ok for 3PD or should we use a global function like `woocommerce_interactivity_use_add_to_cart_store()`?